crates/tuono/src/route.rs

```diff
@@ -261,4 +262,12 @@ mod tests {
             assert_eq!(route.output_file_path(), PathBuf::from(html))
         }
     }
+
+    #[test]
+    fn should_correctly_replace_double_underscore() {
+        let dyn_info = AxumInfo::new(&Route::new("/[posts__id]".to_string()));
```

Comment:
What about also testing the scenario with `dyn_` + `_catch_all` that is more similar to the real bug?